src/drivers/intel/cavs/dmic.c

```diff
@@ -204,31 +188,31 @@ static enum task_state dmic_work(void *data)
 			continue;
 
 		if (dmic->startcount == DMIC_UNMUTE_CIC)
-			dmic_update_bits(dai, base[i] + CIC_CONTROL,
+			dai_update_bits(dai, base[i] + CIC_CONTROL,
 					 CIC_CONTROL_MIC_MUTE_BIT, 0);
 
```

Comment:
You need to align the next line after dai_update_bits just exactly under the first argument. Please also see the checkpatch warnings: https://sof-ci.01.org/sofpr/PR1949/build3554/checkpatch/